datadayessentials/data_retrieval/_base.py

```diff
@@ -147,6 +147,14 @@ def __init__(self, authentication: IAuthentication):
     def load(self) -> Any:
         pass
 
+class IParquetLoader(IDataFrameLoader):
+    def __init__(self, authentication: IAuthentication):
+        pass
+
+    @abstractmethod
+    def load(self) -> Any:
```

Comment:
Guessing it should return an object of DataFrame type as it inherits from dataframeloader?

Reply:
Good catch